index.js

```diff
@@ -362,7 +362,10 @@ class ScmBase {
                 validate(jobList, Joi.array().items(
                     Joi.object().keys({
                         name: Joi.reach(dataSchema.models.job.base, 'name').required(),
-                        ref: Joi.string().required()
+                        ref: Joi.string().required(),
+                        username: Joi.reach(dataSchema.core.scm.pr, 'username'),
```

Comment:
are these by default optional ? Just want to make sure that for scm's that have not implemented these fields yet, won't break on rebuild.

Reply:
yeah, optional by default unless require explicitly